NAMESPACE

```diff
@@ -1,4 +1,5 @@
 useDynLib(RcppSimdJson, .registration=TRUE)
 exportPattern("^[[:alpha:]]+")
+export(.deserialize_json)
```

Comment:
No.  Functions starting with a dot are internal and by default not exported.  We would not export a function with a dot.

One _could_ consider remaning the helper function to make it visible.  @knapply and I once decided to have it hidden (and have the extra safeties in a user-facing function).

Reply:
The point is that it would not be for 'general consumption' but for 'expert' use. There are often so-called technical utility functions starting with a '.'

Reply:
I am suggesting this as the point of RcppSimdJson is that it is fast. It seems a waste to not unleash it's full potential!